ppci/lang/python/python2ir.py

```diff
@@ -526,7 +534,8 @@ def gen_call(self, expr):
         return value
 
     def gen_num(self, expr):
-        num = expr.n
+        # ast.Constant.value / ast.Num.n
+        num = expr.value if hasattr(expr, 'value') else expr.n
```

Comment:
I'd like to modify this line, and move it to the place where the python version specifics are handled. Please modify this, such that this function does not take `expr` but gets `num` passed in.

Reply:
I switched it to take both - `expr` and `num`, to retain the possibility of calling `not_impl` with `expr` (0b5259a6021213489c766f240a3739e9126a0251).
If you feel strongly about it, I can switch it up exactly as you say - let me know.